usr/sbin/p11sak/p11sak.c

```diff
@@ -1235,14 +1429,19 @@ static CK_RV priv_key_print_attributes(CK_SESSION_HANDLE session,
     CK_BBOOL a_never_extractable;
     CK_ULONG bs = sizeof(CK_BBOOL);
 
-    CK_ATTRIBUTE bool_tmplt[] = { { CKA_TOKEN, &a_token, bs }, { CKA_PRIVATE,
-            &a_private, bs }, { CKA_MODIFIABLE, &a_modifiable, bs }, {
-    CKA_DERIVE, &a_derive, bs }, { CKA_LOCAL, &a_local, bs }, {
-    CKA_SENSITIVE, &a_sensitive, bs }, { CKA_DECRYPT, &a_decrypt, bs }, {
-            CKA_SIGN, &a_sign, bs }, { CKA_UNWRAP, &a_unwrap, bs }, {
-    CKA_EXTRACTABLE, &a_extractable, bs }, {
-    CKA_ALWAYS_SENSITIVE, &a_always_sensitive, bs }, {
-    CKA_NEVER_EXTRACTABLE, &a_never_extractable, bs } };
+    CK_ATTRIBUTE bool_tmplt[] = { 
+        { CKA_TOKEN, &a_token, bs }, 
```

Comment:
same comment about the use of sizeof() as above.

Reply:
Done